examples/vision/siamese.py

```diff
@@ -0,0 +1,232 @@
+
+"""
+# Siamese Network with Contrastive Loss
```

Comment:
Suggestion:

```suggestion
# Siamese network with a contrastive loss
```

- Use `Sentence case for titles, descriptions, and section names` in line with most example on https://keras.io/examples/ (including the original ones by @fchollet)